### PR TITLE
[feature] Created default permissions for the default groups #55

### PR DESCRIPTION
**Pull request by ManishShah120**
Fixes: #55

### CODE REVIEW

**Comment by ManishShah120** [comment]
@atb00ker , @nemesisdesign

**Review by nemesisdesign — CHANGES_REQUESTED** [review]
@ManishShah120 some test is failing in the sample app, which is django app with which we test the extensibility of openwisp-ipam (see the README for more info). 

To un the tests for the sample app, run `SAMPLE_APP=1 ./runtests.py`.

**Comment by ManishShah120** [comment]
> @ManishShah120 some test is failing in the sample app, which is django app with which we test the extensibility of openwisp-ipam (see the README for more info).
> 
> To un the tests for the sample app, run `SAMPLE_APP=1 ./runtests.py`.

@nemesisdesign I have resolved the requested changes. Let me know If there is anything more I need to do.

File: openwisp_ipam/tests/test_admin.py [hunk 1]
```diff
@@ -339,3 +340,32 @@ def test_change_view_master_multitenant(self):
...
+            'add_subnet',
+            'change_subnet',
+            'delete_subnet',
+            'view_subnet',
+        ]
+        perms = list(admin.permissions.values_list('codename', flat=True))
+        for p in admin_permissions:
+            self.assertIn(p, perms)
+
+    def test_operator_group_permission(self):
```
Comment (atb00ker) [inline openwisp_ipam/tests/test_admin.py:360]: Is it possible to write another test to ensure that operator does not have permissions that it's not suppose to have?

**Review by atb00ker — COMMENTED** [review]
I completely forgot to review this one, the code looks good.
I'll test it tomorrow.
Meanwhile, can you check the comment below:

**Review by nemesisdesign — COMMENTED** [review]
It was not on the priority board, I added it so we can keep track of it.
I will test it asap. Thanks @ManishShah120.

File: openwisp_ipam/tests/test_admin.py (hunk 1 above)
Comment (ManishShah120) [reply openwisp_ipam/tests/test_admin.py:360]: Okay @atb00ker, I will try to add that too if possible.

Comment (atb00ker) [reply openwisp_ipam/tests/test_admin.py:360]: Great. Other than this, Looks good from my end. Tested. it's working.
Feel free to take another issue when you are done with updating this and waiting for Fed to review / merge it! :smile:

**Review by nemesisdesign — APPROVED** [review]
Thanks @ManishShah120 for your great work and @atb00ker for your great review and testing :+1:

**Pull request merged**